sdk/go/common/resource/plugin/langruntime.go

```diff
@@ -185,10 +185,13 @@ type AboutInfo struct {
 }
 
 type RunPluginInfo struct {
+	Name             string
+	Kind             string
```

Comment:
Note that the new `Name` field here is simply the `prefix`, e.g. `mycomponent (resource)`.